website/assets/scss/_blog.scss

```diff
@@ -0,0 +1,194 @@
+.blog{
+
+  &-rounded {
+    border-radius: .5 * $blog-gutter-xs;
+  }
+
+  &__pagination {
+    margin: $blog-gutter-md;
+
+    &>li>a {
+      &:focus {
+        outline: 0px;
+      }
+    }
+
+    &__active {
+      cursor: initial;
+      font-size: 1rem;
+      font-weight: bold;
+      color: $primary
+    }
+
+    &__page {
+      cursor:pointer;
+      font-size: .85rem;
+      padding: $blog-gutter-xs/5 $blog-gutter-xs;
+    }
+
+    &__previous {
+      &::before {
+        background: url('/images/page_prev@2x.svg') no-repeat center;
+      }
+    }
+
+    &__next {
+      &::before {
+        background: url('/images/page_next@2x.svg') no-repeat center;
+      }
+    }
+
+    &__button {
+      margin: 0 $blog-gutter-xs;
+      cursor:pointer;
+      display: block;
+      width: 36px;
+      height: 36px;
+      border: 1px solid $darker-grey;
+
+      &::before {
+        display: block;
+        content: "";
+        width: 34px;
+        height: 34px;
+      }
+    }
+
+    &__disabled {
+      &>a {
+        cursor:initial;
+        border: 1px solid $dark-grey;
+
+        &::before {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &__post-preview-title {
+    cursor:pointer;
+  }
+
+  &__post-preview-image {
+    cursor:pointer;
+  }
+
+  &__post-title {
+    min-height: 50vh;
+    padding-top: $blog-gutter-xl;
+    padding-bottom: $blog-gutter-xl;
+    margin-bottom: $blog-gutter-xl;
+
+    &>h1 {
+      font-size: 48px;
+    }
+
+    &::after {
+      border-radius: .5 * $blog-gutter-xs;
+    }
+
+    &__back-button {
+      position: absolute;
+      top: $blog-gutter-xl - $blog-gutter-xs;
+    }
+  }
+
+  &__post-image {
+
+    &>img {
+      max-width: 100%;
+    }
+
+    &__center {
+      display: flex;
+      justify-content: center;
+    }
+
+    &__left {
+      float: left;
```

Comment:
Let's use flex or `margin-something: auto` instead of floats. Floats are buggy. 
justify-content: flex-start for left or flex-end for right should do.

Reply:
This in here we need to float image into article paragraphs, rather then style it as a layout? I think float is meant to be used for those cases ?

Reply:
true, it's fine in that case